src/components/WheelPicker/index.tsx

```diff
@@ -97,30 +97,34 @@ export interface WheelPickerProps {
   /**
    * Props to be sent to the FlatList
    */
-  flatListProps?: Partial<FlatListProps<ItemProps>>;
+  flatListProps?: Partial<FlatListProps<ItemProps<T>>>;
 }
 
-const WheelPicker = ({
-  items: propItems,
-  itemHeight = ITEM_HEIGHT,
-  numberOfVisibleRows = 5,
-  activeTextColor = Colors.$textPrimary,
-  inactiveTextColor,
-  textStyle,
-  label,
-  labelStyle,
-  labelProps,
-  onChange,
-  align = WheelPickerAlign.CENTER,
-  disableRTL,
-  style,
-  children,
-  initialValue = 0,
-  separatorsStyle,
-  testID,
-  faderProps,
-  flatListProps
-}: WheelPickerProps) => {
+const WheelPicker = <T extends string | number = number>(props: WheelPickerProps<T>) => {
```

Comment:
I think we can remove the default here: `<T extends string | number>`
Also, we have this type here as well